google-mlkit/src/main/java/de/c24/hg_abstraction/scan/ScanView.kt

```diff
@@ -54,22 +58,50 @@ class ScanView@JvmOverloads constructor(
 
     }
 
+    @androidx.camera.camera2.interop.ExperimentalCamera2Interop
+    @androidx.camera.lifecycle.ExperimentalCameraProviderConfiguration
     private fun bindPreview(activity: Activity,cameraProvider : ProcessCameraProvider) {
 
-        var preview : Preview = Preview.Builder()
+        var preview: Preview = Preview.Builder()
             .build()
 
-        var cameraSelector : CameraSelector = CameraSelector.Builder()
+        var cameraSelector: CameraSelector = CameraSelector.Builder()
             .requireLensFacing(CameraSelector.LENS_FACING_BACK)
             .build()
 
         cameraProvider.unbindAll()
 
         preview.setSurfaceProvider(previewview.surfaceProvider)
 
-        cameraProvider.bindToLifecycle(activity as LifecycleOwner, cameraSelector, preview, setupAnalyzer())
+        val camera = cameraProvider.bindToLifecycle(
```

Comment:
consider using View.postDelayed(Runnable action, long delayMillis) instead of handler, which is more java style. 

But code looks fine to me